userspace/libsinsp/threadinfo.h

```diff
@@ -659,6 +660,10 @@ class SINSP_PUBLIC sinsp_threadinfo : public libsinsp::state::table_entry {
 	bool m_parent_loop_detected;
 	libsinsp::state::stl_container_table_adapter<decltype(m_args)> m_args_table_adapter;
 	libsinsp::state::stl_container_table_adapter<decltype(m_env)> m_env_table_adapter;
+	libsinsp::state::stl_container_table_adapter<
```

Comment:
This is a bit ugly because we need to enforce the specific value unwrapping class (the second entry).